test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -10,6 +10,14 @@
 
 public class UtilsTest {
 
+    @Test
+    public void isAnyNullTest() {
+        assertTrue(Utils.isAnyNull((Object) null));
+        assertTrue(Utils.isAnyNull(null, null));
+        assertTrue(Utils.isAnyNull("abc", 1, null));
+        assertFalse(Utils.isAnyNull(1));
+        assertFalse(Utils.isAnyNull(1, "abc", new Integer(1)));
```

Comment:
Consider splitting the cases up into blocks and commenting to describe their purpose.